modules/dag-l1/src/main/scala/org/tessellation/dag/l1/domain/trust/storage/TrustDbStorage.scala

```diff
@@ -0,0 +1,12 @@
+package org.tessellation.dag.l1.domain.trust.storage
+
+import org.tessellation.schema.peer.PeerId
+import org.tessellation.schema.trust.TrustDbValues
+
+trait TrustDbStorage[F[_]] {
```

Comment:
trait is an interface and it's part of a domain. domain by definition should be "framework-agnostic" as much as possible. that's the idea of abstraction and using interfaces, to make it exchangeable between different implementations. like an algebra and multiple interpreters. that being said, domain should be unaware of "database"